content/accessibility/developer-toolkit.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Developer toolkit
+---
+
+A great way to find answers to specific accessibility questions is to do an online search for the component/issue in question + 'a11y', 'accessible', or 'accessibility', and scan for official websites like W3C ([https://w3.org](https://w3.org)). 
+
+**Internal resources**
+- #accessibility channel in slack
+- Fluent community's accessibility channel in Teams (This channel is dedicated to making Fluent Design System accessible, but is home to many talented designers and developers passionate about accessibility)
+## Accessibility Insights
+One of the best things a developer can do is learn by doing, and the same goes for developing accessibly. 
```

Comment:
This sounds weird, but I'm not sure how to rewrite it rn. Maybe: "As with most other development topics, accessibility is better learnt by doing." 🤔